app/javascript/modCenter/singleArticle/index.jsx

```diff
@@ -31,16 +31,22 @@ export default class SingleArticle extends Component {
     const { articleOpened } = this.state;
     const { id, title, publishedAt, cachedTagList, user, key } = this.props;
     const tags = cachedTagList.split(', ').map((tag) => {
-      return (
-        <span className="mod-article-tag fs-s ff-accent lh-base" key={key}>
-          <span className="article-hash-tag">#</span>
-          {tag}
-        </span>
-      );
+      if (tag) {
+        return (
+          <span className="mod-article-tag fs-s ff-accent lh-base" key={key}>
+            <span className="article-hash-tag">#</span>
+            {tag}
+          </span>
+        );
+      }
     });
 
     const newAuthorNotification = user.articles_count <= 3 ? '👋 ' : '';
 
+    const truncateTitle = (title, limit) => {
```

Comment:
You can use CSS to do this. See https://developer.mozilla.org/en-US/docs/Web/CSS/text-overflow

Reply:
Done 👍 was a bit tricky with grid, TLDR I did this by adding `min-width: 0` the `article-title` grid element.

Reference:
https://stackoverflow.com/questions/44430740/how-do-i-set-text-overflow-to-ellipsis-in-css-grid-layout
https://css-tricks.com/preventing-a-grid-blowout/